Vagrantfile

```diff
@@ -0,0 +1,35 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+# Author: David Manouchehri
+
+Vagrant.configure("2") do |config|
+	config.vm.box = "bento/ubuntu-16.04"
+
+	config.vm.provision "shell", inline: <<-SHELL
+		# apt-get update
+		# DEBIAN_FRONTEND=noninteractive apt-get -y upgrade
+		snap install --classic go
+		# apt-get clean
+	SHELL
+
+	config.vm.provision "docker"
+
+	config.vm.provision "shell", privileged: false, inline: <<-SHELL
+		mkdir -p ~/go/src/github.com/linuxkit/
+		ln -s /vagrant ~/go/src/github.com/linuxkit/linuxkit
+		cd ~/go/src/github.com/linuxkit/linuxkit/src/cmd/linuxkit
+		go get -d
+		go install
+		echo "export PATH=${PATH}:${HOME}/go/bin" >> ~/.bashrc
```

Comment:
Might this be clearer as `${GOPATH}/bin`?

Reply:
I might be wrong, but is `${GOPATH}` even set here? I can check, but not much of a difference anyway.

Reply:
No, $GOPATH is not set.

```bash
vagrant@vagrant:~$ echo $GOPATH

vagrant@vagrant:~$
```

Reply:
It seems the default `$GOPATH` is `$HOME/go`, I didn't realise that (I thought it was something one was required to set). OK then.